src/Akeneo/Tool/Component/Connector/Writer/WriterFactory.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Tool\Component\Connector\Writer;
+
+
+use Box\Spout\Writer\WriterFactory as SpoutWriterFactory;
+use Box\Spout\Writer\WriterInterface;
+
+/**
+ * @author    Pierre Jolly <pierre.jolly@akeneo.com>
+ * @copyright 2021 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
```

Comment:
Can you add a comment and create a TIP ticket? Something like:

```suggestion
  * TIP-xxxx: update Spout library on master in 3.x to avoid the override of the SpoutWriterFactory class
  */
```

Reply:
If you're working on it, no need I guess. Just maybe a `TODO pullup: skip this file because...`